merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/http/ResponseSerializers.java

```diff
@@ -152,11 +152,12 @@ public static JsonValue serializeSimulatedActivity(final SimulatedActivity simul
     return Json
         .createObjectBuilder()
         .add("type", simulatedActivity.type)
-        .add("parameters", serializeArgumentMap(simulatedActivity.parameters))
+        .add("parameters", serializeArgumentMap(simulatedActivity.arguments))
```

Comment:
I stopped short of making this change, since it affects the API. I do think we'll want it though, unless there are any objections.

```suggestion
        .add("arguments", serializeArgumentMap(simulatedActivity.arguments))
```

Reply:
gosh, please, yes! @camargo is the arbiter here IMHO

Reply:
Yes I fully support this change. It will require a change in the UI here: https://github.com/NASA-AMMOS/aerie-ui/blob/develop/src/utilities/requests.ts#L1118
Notice how we are already changing the property name to `arguments` as that is what the rest of the UI expects :)

Reply:
@mattdailis Let me know if you are going to change it and I will submit and link a quick PR

Reply:
@camargo awesome, I'll make the change now :+1:

Reply:
Here is the PR: We can merge it once this PR is ready and merged: https://github.com/NASA-AMMOS/aerie-ui/pull/2